### ISSUE TITLE
Figure for continuous SC sampler with hyetograph

### ISSUE THREAD

**Issue by jds485** [USGS-R/drb-inland-salinity-ml#184]
y-axis 1: Daily SC from a continuous sampler.
y-axis 2 (inverted): precipitation hyetograph. Bars/points colored by avg. temperature (< 0 C, > 0 C)
x-axis: 1 year of data with at least one winter snow signal. 
Maybe 2021 water year for site 01436655

**Comment by lekoenig** [comment]
USGS site [`01436655` ](https://waterdata.usgs.gov/nwis/inventory?agency_code=USGS&site_no=01436655) is not included in our target data because it's not located on the NHM/PRMS fabric in `p1_reaches_sf`. If we want to plot this site, we'll have to find the nearest segment and grab gridmet data from that segment to use for the hyetograph. Alternatively, are there certain characteristics of 01436655 that make it a good candidate to plot?

As a proof of concept, I tried making this plot for site [`01467000`](https://waterdata.usgs.gov/monitoring-location/01467000/#:~:text=Monitoring%20location%2001467000%20is%20associated,to%201921%20are%20available%20online.&text=A%20separate%20tab%20will%20open%20with%20the%20requested%20data.) for water year 2021. It's a little hacky, and I had to use another plotting helpers package, [`{cowplot}`,](https://cran.r-project.org/web/packages/cowplot/vignettes/introduction.html) to overlay the SC time series and the hyetograph. Here, the SC time series comes from `p2_daily_combined` and the precip data comes from `p1_gridmet`. I don't think the `pr` variable in the gridmet data captures snowfall (e.g. see the plot in [this](https://github.com/USGS-R/drb-inland-salinity-ml/pull/19#issue-1067602428) PR for the same site), but is this roughly what you had in mind?

![draft_SC_ts_w_hyetograph_plot](https://user-images.githubusercontent.com/8785034/203609498-b2f16103-b472-4a76-a950-7f75a691edd6.png)

**Comment by jds485** [comment]
For the snowfall figure, I used this figure from you in a presentation. It's the same site, but I think the SC timeseries is hourly. I'm wondering about the precip/snowfall differences. The plot in the PR has 2" max precip, but this one has 5".

![image](https://user-images.githubusercontent.com/8761242/203612377-7479e15d-3be7-4406-b547-0a88fbc0a5d4.png)

**Comment by jds485** [comment]
I really like this site as an example, though. It captures both precip causing a decrease in SC (non-winter) and increase in SC (winter). Coloring by the (min+max)/2 temperature could help to see those differences

**Comment by lekoenig** [comment]
> It's the same site, but I think the SC timeseries is hourly. I'm wondering about the precip/snowfall differences. The plot in the PR has 2" max precip, but this one has 5". Timing is also different (many more points in new plot, even in winter).

OK, yeah, when I've plotted this site before I used the "raw" instantaneous data and manually pulled snowfall accumulation data from the NOAA daily climate summaries (GHCND network). In my plot above I used the daily average SC and was pulling precip from our gridmet data, which I don't _think_ includes snowfall accumulation. So the reason the max precip axes are different is because they're coming from different data sources which may capture different things (rainfall vs snowfall).

**Comment by jds485** [comment]
Okay, thanks!

> You're referring to coloring the SC line by mean temp, right?

I was thinking the hyetograph because that would indicate likely snow vs. rain, but that's assuming snow is captured by gridMET. I'll search for an answer to this because it's important for this work. Coloring the SC might stand out better than the hyetograph, though. Open to either method

**Comment by lekoenig** [comment]
> I'll search for an answer to this because it's important for this work.

Yeah, I completely agree. I hadn't really considered or assessed how gridmet treats snowfall. @msleckman, have you seen any gridmet documentation related to snow?

**Comment by jds485** [comment]
Let's make a new issue to add a SWE dataset. Not a current need. I'd like to see how models are performing in winter with the current met dataset before adding a new dataset.
